media/transcoder/job_test.py

```diff
@@ -242,6 +257,96 @@ def test_create_job_with_animated_overlay(capsys, test_bucket):
     assert "Deleted job" in out
 
 
+def test_create_job_with_set_number_spritesheet(capsys, test_bucket):
+    create_job_with_set_number_images_spritesheet.create_job_with_set_number_images_spritesheet(
+        project_id,
+        location,
+        input_uri,
+        output_uri_for_set_number_spritesheet,
+    )
+    out, _ = capsys.readouterr()
+    job_name_prefix = f"projects/{project_number}/locations/{location}/jobs/"
+    assert job_name_prefix in out
+
+    str_slice = out.split("/")
+    job_id = str_slice[len(str_slice) - 1].rstrip("\n")
+    job_name = f"projects/{project_number}/locations/{location}/jobs/{job_id}"
+    assert job_name in out
+
+    get_job.get_job(project_id, location, job_id)
+    out, _ = capsys.readouterr()
+    assert job_name in out
```

Comment:
Does asserting the job_name actually men it was successful? Should we be asserting some kind of "succeeded" message in the response instead?

Reply:
Added comment to explain this